src/varlinkctl/varlinkctl.c

```diff
@@ -376,9 +376,9 @@ static int verb_call(int argc, char *argv[], void *userdata) {
         arg_json_format_flags &= ~JSON_FORMAT_OFF;
 
         if (parameter) {
-                r = json_parse_with_source(parameter, "<argv[4]>", 0, &jp, &line, &column);
+                r = json_parse_with_source(parameter, "<argv[3]>", 0, &jp, &line, &column);
```

Comment:
oh, this is actually intended that way. because the "verb" logic jumps over the `argv[0]` item in the `argv[]` when it dispatches things to the verb implementations. But here we probably should output the "official" array index, as seen from the shell.

maybe we should add a comment here though

Reply:
Ah, indeed...